<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-AJRJQFP7" name="com.testRunner.TestRunnerTestNG" tests="4" failures="0" timestamp="06 Jul 2020 03:38:03 GMT" time="15.329" errors="3">
  <testcase name="runScenario" time="0.339" classname="com.testRunner.TestRunnerTestNG">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.utilityMethods.UtilityClass.implicitWait(UtilityClass.java:245)
	at com.stepDefinition.Hooks.ticketGeneration(Hooks.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at cucumber.runtime.Utils$1.call(Utils.java:26)
	at cucumber.runtime.Timeout.timeout(Timeout.java:16)
	at cucumber.runtime.Utils.invoke(Utils.java:20)
	at cucumber.runtime.java.JavaHookDefinition.execute(JavaHookDefinition.java:65)
	at cucumber.runner.HookDefinitionMatch.runStep(HookDefinitionMatch.java:16)
	at cucumber.runner.TestStep.executeStep(TestStep.java:65)
	at cucumber.runner.TestStep.run(TestStep.java:50)
	at cucumber.runner.TestCase.run(TestCase.java:42)
	at cucumber.runner.Runner.runPickle(Runner.java:50)
	at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:79)
	at cucumber.api.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:24)
	at cucumber.api.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="14.248" classname="com.testRunner.TestRunnerTestNG"/>
  <testcase name="runScenario" time="0.372" classname="com.testRunner.TestRunnerTestNG">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElements(DefaultElementLocator.java:85)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementListHandler.invoke(LocatingElementListHandler.java:36)
	at com.sun.proxy.$Proxy20.get(Unknown Source)
	at com.stepDefinition.StepDefinition.user_has_to_click_the_select_button(StepDefinition.java:82)
	at ✽.User has to click the select button "User clicked the select button"(file:src/test/resources/Feature1.feature:31)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="0.370" classname="com.testRunner.TestRunnerTestNG">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElements(DefaultElementLocator.java:85)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementListHandler.invoke(LocatingElementListHandler.java:36)
	at com.sun.proxy.$Proxy20.get(Unknown Source)
	at com.stepDefinition.StepDefinition.user_has_to_feed_the_location(StepDefinition.java:42)
	at ✽.User has to feed the location "London"(file:src/test/resources/Feature1.feature:16)
]]>
    </error>
  </testcase> <!-- runScenario -->
</testsuite> <!-- com.testRunner.TestRunnerTestNG -->
